### DISCUSSION TITLE
Stripe: pass email address for non account holder

### DISCUSSION THREAD

**Discussion by jeverington** [pretix/pretix#3621] [Ideas]
![Screenshot 2023-09-27 at 15 27 27](https://github.com/pretix/pretix/assets/1991914/2e635709-6108-438d-9d11-024603c9314e)

When this information is collected by an event then it should be passed to Stripe to help with Fraud Detection. Not sure if this works if the customer has an account, but we are not using the customer account functionality.

**Comment by raphaelm** [comment]
We currently never pass personal data to a payment provider that is not strictly required in the spirit of "maximum privacy by default". So if at all, this would need to be an opt-in change and I'm not sure it's worth that (we try not to have even more checkboxes for everything)

**Reply by jeverington** [reply]
Thanks for the response. Yep, I totally get the privacy-first approach, but it would be nice to have the option, especially if you are trying to use the fraud detection tools in Stripe.
